### DISCUSSION TITLE
Error while duplicating a part

### DISCUSSION THREAD

**Discussion by Pixeltrace-GER** [inventree/InvenTree#4609] [Q&A] [answered]
I'm not able anymore to duplicate a part. I'm running in a strange error I can't handle alone.

**My Setup**
```
InvenTree-Version: 0.10.1
Django Version: 3.2.16
Commit Hash: c1f15ef
Commit Date: 25-02-2023
Database: postgresql
Debug-Mode: False
Deployed using Docker: True
Active plugins: False
```

**My error**
```
inventree-db      | 2023-04-15 09:18:46.609 UTC [1560] ERROR:  duplicate key value violates unique constraint "part_partparameter_part_id_template_id_74280ad3_uniq"
inventree-db      | 2023-04-15 09:18:46.609 UTC [1560] DETAIL:  Key (part_id, template_id)=(590, 1) already exists.
inventree-db      | 2023-04-15 09:18:46.609 UTC [1560] STATEMENT:  INSERT INTO "part_partparameter" ("part_id", "template_id", "data") VALUES (590, 1, '0') RETURNING "part_partparameter"."id"
inventree-server  | An error occurred in the current transaction. You can't execute queries until the end of the 'atomic' block.
```

Any chance to help with this information?

**Comment by SchrodingersGat** [comment, marked as answer]
This looks to be a duplicate issue of https://github.com/inventree/InvenTree/issues/4602 which has been fixed in the upcoming stable release.

Can you please check on the [demo server](https://demo.inventree.org) to determine if (from your perspective) the issue has been addressed?

**Reply by Pixeltrace-GER** [reply]
Info at me: Check next time the issues...

It looks like the same Bug. On the demo server it is all fine.

**Reply by SchrodingersGat** [reply]
Not many pending issues before the next release, so should be out pretty soon
